css-brochure-form/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h3>LEARN MORE</h3>
+    <div class="container">
+      <div>
+        <label for="email">COMPANY EMAIL</label>
+        <input type="text" id="email" placeholder="john@company.co" />
+      </div>
+      <div>
+        <label for="name">NAME</label>
+        <input type="text" id="name" placeholder="John Smith" />
+      </div>
+      <div>
+        <label for="location">WHERE ARE YOU HIRING?</label>
+        <input type="text" id="location" placeholder="City, Country" />
+      </div>
+      <div>
+        <label for="hire"
+          >HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label
+        >
```

Comment:
Not sure what happened here, but your indentation got a bit wacky 😬

Reply:
Yeah, that's been happening after I save a file I think. Not too sure what's causing it but I'll try and find out.